modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,90 @@
+
+* The following commands are deprecated and will be removed in the next release:
+** `tkn pipeline create`
+** `tkn task create`
+
+* The `-f` flag with the `tkn resource create` command is now deprecated. It will be removed in the next release.
+
+* The `-t` flag and the `--timeout` flag (with seconds format) for the `tkn clustertask create` command are now deprecated. Only duration timeout format is now supported, for example `1h30s`. These deprecated flags will be removed in the next release.
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+* If you are upgrading from an older version of {pipelines-title}, you will need to delete your existing deployments before upgrading to {pipelines-title} version 1.0.
+* Submitting the same `v1alpha1` Tasks more than once will result in an error. Use `oc replace` instead of `oc apply` when re-submitting a `v1alpha1` Task.
+
+[id="fixed-issues-1-0_{context}"]
+== Fixed issues
+* link:https://issues.redhat.com/browse/SRVKP-637[SRVKP-637] : Previously, the `DeploymentConfig` Task triggered a new deployment build even when an image build was already in progress. This caused the deployment of the Pipeline to fail. With this fix, the `deploy task` command  is now replaced with the `oc rollout status` command which waits for the in-progress deployment to finish.
+* link:https://issues.redhat.com/browse/SRVKP-631[SRVKP-631]: Support for `APP_NAME` parameter is now added in Pipeline templates.
+* link:https://issues.redhat.com/browse/SRVKP-616[SRVKP-616]: Previously, the Pipeline template for Java S2I failed to look up the image in the registry. With this fix, the image will be looked up using the existing image PipelineResources instead of the user provided `IMAGE_NAME` parameter.
+* link:https://issues.redhat.com/browse/SRVKP-383[SRVKP-383]: All the Openshift Pipelines images are now based on the Red Hat Universal Base Images (UBI).
+* link:https://issues.redhat.com/browse/SRVKP-700[SRVKP-700]:  Previously, when the Pipeline was installed in the namespace other than `tekton-pipelines`, the`tkn version` command displayed the Pipeline version as `unknown`. With this fix, the `tkn version` command now displays the correct Pipeline version in any namespace.
+* link:https://issues.redhat.com/browse/SRVKP-698[SRVKP-698]: The `tkn version -c` command is now modified to not show newer versions of the CLI available upstream.
+* link:https://issues.redhat.com/browse/SRVKP-695[SRVKP-695]: Non-admin users can now list the ClusterTriggerBindings.
+* link:https://issues.redhat.com/browse/SRVKP-693[SRVKP-693]: The EventListener CompareSecret function is now fixed for the CEL Interceptor.
+* link:https://issues.redhat.com/browse/SRVKP-690[SRVKP-690]: The `list`, `describe`, and `start` sub-commands for `task` and `clustertask` now correctly display the output in case a Task and ClusterTask have same names.
```

Comment:
I think the issue was only with the describe command.. @piyush-garg please clarify

Reply:
It is with
 - tkn task describe
 - tkn clustertask describe
 - tkn taskrun list <taskname>
 - tkn task start --last
 - tkn clustertask start --last 

This is on top of my head, maybe I added the fix somewhere else also. :smile: